website/components/navigation/Navigation.tsx

```diff
@@ -19,57 +23,73 @@ const logoStyles = css`
 `;
 
 function Navigation() {
+  const viewport = useViewportSize();
+  const isMobile = viewport && viewport.width < 768;
```

Comment:
`!!viewport` to keep `isMobile` to a boolean

Reply:
👍